app/models/profile.rb

```diff
@@ -163,6 +163,20 @@ def deactivate_for_gpo_verification
     update!(active: false, gpo_verification_pending_at: Time.zone.now)
   end
 
+  def deactivate_for_in_person_verification_cancelled
+    transaction do
+      deactivate(:verification_cancelled)
```

Comment:
We shouldn't be calling deactivate on pending profiles. I see that the existing code does set active to false. I think we should wait with this until in_person_verification_pending_at is fully implemented, and then we can take a look at whether we can stop "deactivating" pending profiles that haven't been activated.